src/framework/ui/CMakeLists.txt

```diff
@@ -57,7 +57,10 @@ else()
     )
 endif()
 
+qt_wrap_ui(GENERATED_UI_HEADERS ${CMAKE_CURRENT_LIST_DIR}/dev/testdialog.ui)
```

Comment:
This should not be necessary, because we have AUTOUIC. Just including ui_testdialog.h will be enough.

Reply:
for some reason AUTOUIC doesn't work :(